test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -100,21 +110,22 @@ public static IEnumerable<object[]> MessageSizesData
         {
             get
             {
-                yield return new object[] { new string('A', 5 * 1024)};
-                yield return new object[] { new string('A', 5 * 1024 * 1024 + 32)};
+                yield return new object[] { new string('A', 5 * 1024), new WebSocketsTransport()};
+                yield return new object[] { new string('A', 5 * 1024 * 1024 + 32), new WebSocketsTransport() };
+                yield return new object[] { new string('A', 5 * 1024), new LongPollingTransport(new HttpClient())};
```

Comment:
This does not seem right (given the name of the property). If needed at all (and I am not convinced it is) you should create a getter that creates a Cartesian product of what before was `MessageSizesData` and `Transports`